src/Sensor/MemoryMeasurement.java

```diff
@@ -7,18 +7,22 @@
 public class MemoryMeasurement extends Measurement {
     
     MemoryMeasurement(Sigar _sigar){
-        super("Memory measure");
-        sigar = _sigar;
+        super("Memory measure", _sigar);
     }
 
     public String getActualMeasure(){
-        Long value = (long) -1;
+        Long memoryUsageValueBytes = (long) -1;
+        Long memoryUsageValueMegabytes = (long) -1;
```

Comment:
((Not an issue, topic for discussion))
Wouldn't be better to move this declaration to _try_ scope, return result at the end of this scope and return some constant error indication at the end of method?